### ISSUE TITLE
Compatibility with HD editions

### ISSUE THREAD

**Issue by JeodC** [eglaysher/rlvm#99]
Clannad HD is able to run in rlvm straight from Steam. Side Stories has a separate issue, but Kanon HD and Planetarian HD currently can't run because the data structure is different. There is no Gameexe.ini present, and if there is, it's packed into a `.PAK` file that needs unpacking and/or decoding.

Since these are the only commercially available editions out there (that I could find), it would be neat to get them running or at least document them to explain why they're incompatible.

They're also more controller friendly in the sense that menus are rendered within the game, not the system. Mouse emulation via joystick works well with newer reallive, not so much the older versions.

**Comment by Mickey-snow** [comment]
They're incompatible because they use Siglus, a different engine.

**Comment by JeodC** [comment]
> They're incompatible because they use Siglus, a different engine.

Yes. This ticket is a question of compatibility, if rlvm will extend support. I assume not.

**Comment by NintendoManiac64** [comment]
As I mentioned in #91, this is presumably more of for completionist academic interest rather than a usability in the context of wine's modern-day compatibility of [insert current year of the Linux desktop]? I say this because I'd been running unsupported VNs like Tomoyo After via wine for a few years now, and I know that Side Stories is Steam Deck verified as well.
